model.py

```diff
@@ -0,0 +1,58 @@
+from app import db
+
+
+class Bookmakers(db.Model):
```

Comment:
model class names should be in singular - Bookmaker, table names in plural - bookmakers, beacause an object of this class will contain only one bookmaker, table in database will store all bookmakers